server/app/services/geo/AddressAttributes.java

```diff
@@ -0,0 +1,35 @@
+package services.geo;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class AddressAttributes {
```

Comment:
suggestion (non-blocking): the plural name suggests a collection, maybe something like `BasicAddress`?